src/devices/opus_xt300.c

```diff
@@ -33,15 +33,18 @@ CC: check sum (simple sum) except 0xFF preamble
 static int opus_xt300_callback(r_device *decoder, bitbuffer_t *bitbuffer)
 {
     int ret = 0;
+    int fail_code = 0;
     int row;
     int chk;
     uint8_t *b;
     int channel, temp, moisture;
     data_t *data;
 
     for (row = 0; row < bitbuffer->num_rows; row++) {
+        fprintf(stderr, "%s: bits_per_row %d\n", __func__, bitbuffer->bits_per_row[row]);
```

Comment:
This debug fprintf should be commented out?

Reply:
opps..

Reply:
fixed